src/System.Net.Http/tests/FunctionalTests/PostScenarioUWPTest.cs

```diff
@@ -0,0 +1,163 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System;
+using System.Diagnostics;
+using System.IO;
+using System.Net.Http;
+using System.Text;
+using System.Threading.Tasks;
+
```

Comment:
The coding guildelines used to have an explicit rule that said to separate top-level group of 'using' statements with a newline.

However, I looked in the document and I don't see that anymore. And the example that is shown doesn't have a newline anymore.
https://github.com/dotnet/corefx/blob/master/Documentation/coding-guidelines/coding-style.md

@stephentoub Do you know when those style rules changed?